packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react';
+import { ChatListItem } from '../ChatListItem/ChatListItem';
+import { SampleChats } from '../ChatListItem/sampleData';
+import { FluentProvider, webLightTheme } from '@fluentui/react-components';
+import { FluentThemeProvider } from '@azure/communication-react';
+import { FluentTheme } from '@fluentui/react';
+import { Chat as GraphChat } from '@microsoft/microsoft-graph-types';
+import { ProviderState, Providers } from '@microsoft/mgt-element';
+
+// this is a stub to move the logic here that should end up here.
+export const ChatList = (chatSelected: { chatSelected: (e: GraphChat) => void }) => {
+  const [myId, setMyId] = useState<string>();
+
+  useEffect(() => {
+    const currentUserId = () => getCurrentUser()?.id.split('.')[0] || '';
+
+    const getMyId = async () => {
```

Comment:
Is this temp or does setting the userId via state matter for some reason?

Reply:
It can be temporary I suppose, we just need the Id to be passed to ChatListItem, and this was an established way to obtain it, and be reactive to changes to it (via sign out).